forage-android/src/main/java/com/joinforage/forage/android/ui/FormatPanTextWatcher.kt

```diff
@@ -79,6 +82,36 @@ fun getTransformedPosition(pos: Int, oldPrettyText: String, newPrettyText: Strin
     return minOf(shiftedPos, newPrettyText.length)
 }
 
+fun getThemeAccentColor(context: Context): Int {
+    val outValue = TypedValue()
+    context.theme.resolveAttribute(android.R.attr.colorAccent, outValue, true)
+    return outValue.data
+}
+
+fun TypedArray.getBoxCornerRadiusBottomStart(boxCornerRadius: Float): Float {
+    val boxCornerRadiusBottomStart =
+        getDimension(com.joinforage.forage.android.R.styleable.ForagePANEditText_boxCornerBottomStart, 0f)
+    return if (boxCornerRadiusBottomStart == 0f) boxCornerRadius else boxCornerRadiusBottomStart
+}
+
+fun TypedArray.getBoxCornerRadiusTopEnd(boxCornerRadius: Float): Float {
+    val boxCornerRadiusTopEnd =
+        getDimension(com.joinforage.forage.android.R.styleable.ForagePANEditText_boxCornerTopEnd, 0f)
+    return if (boxCornerRadiusTopEnd == 0f) boxCornerRadius else boxCornerRadiusTopEnd
+}
+
+fun TypedArray.getBoxCornerRadiusBottomEnd(boxCornerRadius: Float): Float {
+    val boxCornerRadiusBottomEnd =
+        getDimension(com.joinforage.forage.android.R.styleable.ForagePANEditText_boxCornerBottomEnd, 0f)
+    return if (boxCornerRadiusBottomEnd == 0f) boxCornerRadius else boxCornerRadiusBottomEnd
+}
+
+fun TypedArray.getBoxCornerRadiusTopStart(boxCornerRadius: Float): Float {
+    val boxCornerRadiusTopStart =
+        getDimension(com.joinforage.forage.android.R.styleable.ForagePANEditText_boxCornerTopStart, 0f)
+    return if (boxCornerRadiusTopStart == 0f) boxCornerRadius else boxCornerRadiusTopStart
+}
+
```

Comment:
Are these related to the FormatPanTextWatcher? If not, can we create a new file for these methods?

Reply:
Can we also make sure that internal-facing code uses the `internal`  access modifier. Currently, these methods may default to `public`

Reply:
Updated. Thanks!